README.md

```diff
@@ -193,3 +200,134 @@ For more information on running and tearing down k8s environments, follow this [
+```
+
+*Note:* If your IP address changes you will need to update the ```/etc/hosts``` entry for localhost-k8s.
+
+### 3. Deploy the DBP
+
+```bash
+helm install alfresco-incubator/alfresco-dbp \
+--set alfresco-infrastructure.alfresco-api-gateway.keycloakURL="http://localhost-k8s/auth/" \
+--set alfresco-infrastructure.rabbitmq-ha.enabled=false \
+--set alfresco-infrastructure.alfresco-activiti-cloud-registry.enabled=false \
+--set alfresco-infrastructure.alfresco-api-gateway.enabled=false \
+--set alfresco-content-services.externalHost="localhost-k8s" \
+--set alfresco-content-services.networkpolicysetting.enabled=false \
+--set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://localhost-k8s/auth" \
+--set alfresco-content-services.repository.replicaCount=1 \
+--set alfresco-content-services.repository.livenessProbe.initialDelaySeconds=420 \
+--set alfresco-content-services.pdfrenderer.livenessProbe.initialDelaySeconds=300 \
+--set alfresco-content-services.libreoffice.livenessProbe.initialDelaySeconds=300 \
+--set alfresco-content-services.imagemagick.livenessProbe.initialDelaySeconds=300 \
+--set alfresco-content-services.share.livenessProbe.initialDelaySeconds=420 \
+--set alfresco-content-services.repository.resources.requests.memory="2000Mi" \
+--set alfresco-content-services.pdfrenderer.resources.requests.memory="500Mi" \
+--set alfresco-content-services.imagemagick.resources.requests.memory="500Mi" \
+--set alfresco-content-services.libreoffice.resources.requests.memory="500Mi" \
+--set alfresco-content-services.share.resources.requests.memory="1000Mi" \
+--set alfresco-content-services.postgresql.resources.requests.memory="500Mi" \
+--set alfresco-process-services.processEngine.environment.IDENTITY_SERVICE_AUTH="http://localhost-k8s/auth" \
+--set alfresco-process-services.processEngine.environment.IDENTITY_SERVICE_ENABLED=true \
+--set alfresco-process-services.processEngine.resources.requests.memory="1000Mi" \
+--set alfresco-process-services.adminApp.resources.requests.memory="250Mi"
+--namespace $DESIREDNAMESPACE
+```
+
+### 4. Check deployment status of DBP
+
+```bash
+kubectl get pods --namespace $DESIREDNAMESPACE
+```
+
+*Note:* When checking status, your pods should be ```READY 1/1``` and ```STATUS Running```
+
+### 5. Check DBP Components
+
+You can access DBP components at the following address
+- http://localhost/alfresco
+- http://localhost/share
+- http://localhost/activiti-app
+- http://localhost/activiti-admin
+- http://localhost/auth/
+
+*Notes:*
+
+- As deployed, the activiti-app starts in read-only mode.  
+  - Apply a license by uploading an Activiti license file after deployment.
+
+- As deployed, the activit-admin app does not work because it is not configured with the correct server endpoint. 
+  - To fix that, click 'Edit endpoint configuration' and then in the form enter http://localhost-k8s for the server address.
```

Comment:
Do we have a JIRA issue to track resolving this?

Reply:
@rgauss Is the JIRA issue against us or activiti? Is that that we need to provide a way to configure a property that is already configurable by activiti or is it that it is not configurable at all?

Reply:
Well this would be APS, and I'm not sure if the current APS deployment supports changing that endpoint configuration through an environment variable or Helm values file.

If it does not, then we should also raise an issue in that backlog and the DBP deployment issue would be dependent on it and only involve setting the value.

Reply:
@rgauss I think that the issue is with https://github.com/Alfresco/alfresco-process-services-deployment/blob/master/helm/alfresco-process-services/values.yaml#L73.  I don't think that that docker container is picking it up. I tried adding ```--set alfresco-process-services.adminApp.environment.ACTIVITI_ADMIN_REST_APP_HOST="http://localhost-k8s"```. It didn't work. It might be with how our chart is passing it or how they are picking it up. I think that we should start with our chart first.  I'll open the bug there.

Reply:
https://issues.alfresco.com/jira/browse/DEPLOY-595